lib/smart_city/organization.ex

```diff
@@ -29,6 +29,8 @@ defmodule SmartCity.Organization do
             orgTitle: nil,
             version: "0.1"
 
+  use Accessible
```

Comment:
Is this use erroneous?  I'm not seeing any new function calls that could be attributed to Accessible

Reply:
It's not erroneous- although it's currently not being used. Tim suggested leaving this in here as future-proofing.